e2e/main.e2e.spec.ts

```diff
@@ -134,16 +135,15 @@ test.describe.serial('Main App Test', () => {
 
 /**
  * Navigate to a specific tab
- * @param path
+ * @param tab
  */
-async function navigateTo(path: string) {
+async function navigateTo(tab: string) {
   try {
     return await Promise.all([
-      page.click(`.nav li[item="/${ path }"] a`),
-      page.waitForNavigation({ url: `**/${ path }`, timeout: 60_000 }),
-      page.screenshot({ path: `${ defaultReportFolder }${ path }-screenshot.png` })
+      page.click(`.nav li[item="/${ tab }"] a`),
+      page.waitForNavigation({ url: `**/${ tab }`, timeout: 60_000 }),
```

Comment:
Shouldn't we wait for the `click` to return before `waitForNavigation` is called? (That is: why do we need this to be a `Promise.all` instead of just `await… await… await…`?)

Reply:
This method is how PW recommends and works fine in order to avoid racing conditions. https://playwright.dev/docs/api/class-page#page-wait-for-navigation